samples/spring-customer-registry-views-quickstart/pom.xml

```diff
@@ -146,7 +146,7 @@
           <arguments>
             <argument>-classpath</argument>
             <classpath/>
-            <argument>-Dlogback.configurationFile=logback-dev-mode.xml</argument>
+            <argument>-Dlogging.config=src/main/resources/logback-dev-mode.xml</argument>
```

Comment:
I think this file `logback-dev-mode.xml` is missing in all spring samples.

Reply:
It was added on https://github.com/lightbend/kalix-jvm-sdk/pull/1449 .. in main now I believe.